test/jdk/javax/swing/JFileChooser/HTMLFileName.java

```diff
@@ -0,0 +1,127 @@
+/*
+ * Copyright (c) 2025, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+import javax.swing.Icon;
+import javax.swing.JFileChooser;
+import javax.swing.JFrame;
+import javax.swing.filechooser.FileSystemView;
+import java.io.File;
+import java.util.List;
+
+/*
+ * @test
+ * @bug 8139228
+ * @summary JFileChooser should not render Directory names in HTML format
+ * @library /java/awt/regtesthelpers
+ * @build PassFailJFrame
+ * @run main/manual HTMLFileName
+ */
+
+public class HTMLFileName {
+
+    public static void main(String[] args) throws Exception {
+        String INSTRUCTIONS = """
+                1. FileChooser shows up a virtual directory and file with name
+                   "<html><h1 color=#ff00ff><font face="Comic Sans MS">Testing Name".
+                2. On "HTML disabled" frame :
+                      a. Verify that the folder and file name must be plain text.
+                      b. If the name in file pane window and also in directory
+                         ComboBox remains in plain text, then test PASS.
+                         If it appears to be in HTML format with Pink color,
+                         then test FAILS.
+                         (Verify for all Look and Feel).
+                3. On "HTML enabled" frame :
+                      a. Verify that the folder and file name remains in HTML
+                         format with name "Testing Name" pink in color.
+                      b. If the name in file pane window and also in directory
+                         ComboBox remains in HTML format string, then test PASS.
+                         If it appears to be in plain text, then test FAILS.
+                         (Verify for all Look and Feel).
+                 """;
```

Comment:
```suggestion
                """;
```
Avoid trailing space, it's not needed.

Reply:
Updated.